tools/build_config/Jenkinsfile

```diff
@@ -273,10 +295,7 @@ pipeline {
       steps {
         script {
           if (isUnix()) {
-            sh '''
-              module load cmake/\$CMAKE_VERSION &&
-              ./tools/build_config/build.sh --package
-            '''
+            sh './tools/build_config/build.sh --package'
```

Comment:
does this not require the CMAKE_VERSION set? package calls CPack which is part of CMake

Reply:
I'm also suspicious whether this will work without the `module load`

Reply:
Didn't realise it'd gone. Whoops.